regions/core/core.py

```diff
@@ -161,6 +160,32 @@ def to_shapely(self):
         """
         raise NotImplementedError("")
 
+    @abc.abstractmethod
+    def make_patch(self, **kwargs):
+        """Convert to mpl patch
+
+        Returns
+        -------
+        patch : `~matplotlib.patches.Patch`
+            Matplotlib patch
+        """
+        raise NotImplementedError
+
+    def plot(self, ax=None, **kwargs):
+        """
+        Calls make_patch method forwarding all kwargs and adds patch
+        to given axis.
+
+        Parameters
+        ----------
+        ax : `~matplolib.axes`, optional
```

Comment:
It can be optional I think (at least it was in photutils), but then it should be defined with e.g. `` ax = plt.gca()`` when not given as parameter.